guacamole-docker/bin/build-guacamole.sh

```diff
@@ -135,6 +136,15 @@ if [ -f extensions/guacamole-auth-openid/target/guacamole-auth-openid*.jar ]; th
     cp extensions/guacamole-auth-openid/target/guacamole-auth-openid*.jar "$DESTINATION/openid"
 fi
 
+#
+# Copy TOTP auth extension if it was built
```

Comment:
Why would it not have been built? From what I recall, the only extension that's conditional within the build is RADIUS support.

Reply:
I took exactly what is done above and below for other extensions :)